docs/core/windows-prerequisites.md

```diff
@@ -13,45 +13,125 @@ ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 
 # Prerequisites for .NET Core on Windows
 
-This article shows you what dependencies you need to deploy and run .NET Core applications on Windows machines and develop using Visual Studio.
+This article shows the needed dependencies to develop .NET Core applications on Windows. The supported OS versions and dependencies that follow apply to the three ways of developing .NET Core apps on Windows:
 
-## Supported Windows versions
+* [Command line](tutorials/using-with-xplat-cli.md)
```

Comment:
I wouldn't leave blank spaces in between lines (if needed, add 3 spaces after each line to make the items show up in separate lines)